spec/computable/registry/challenge.spec.ts

```diff
@@ -311,4 +311,22 @@ describe('Registry: Challenge', () => {
     expect(revealAfter).toBe(true)
   })
 
+  it('Registry can be waitlisted after application stage length.', async () => {
```

Comment:
waitlisted? i think you meant whitelisted

Reply:
Ooops, will fix